text/0025-purchase-order.md

```diff
@@ -599,11 +610,9 @@ version of the purchase order.
 Validation Requirements:
 - The signer must be a Pike agent
 - The signing agent must have a Pike role with the `can-create-po` permission
-- The `po_uid` must refer to an existing purchase order
-- The `version_id` must not already exist as a version on the purchase order
-- The `org_id` must exist in Pike for it be a valid transaction
-- All fields marked `required` in the `CreatePurchaseOrderPayload` must be
-supplied
+- The `uid` must not refer to an existing purchase order
```

Comment:
I think it makes sense to have this as just `uid` since it is that attribute of this "purchase order" struct and the other objects are using it to refer to the purchase order, so it makes sense to have `po_uid` in the other messages as a sort of namespace for what the uid refers to.